mockito/invocation.py

```diff
@@ -177,14 +178,18 @@ def thenRaise(self, *exceptions):
             self.__then(Raise(exception))
         return self
 
+    def thenAnswer(self, *callables):
+        for callable in callables:
+            self._AnswerSelector__then(ReturnAnswer(self.invocation.mock, callable))
```

Comment:
The `_AnswerSelector__then` python extravaganza shouldn't be necessary. `__then` should just work.